### PR TITLE
devel: update image builder link to point to UI

### PR DESCRIPTION
**Pull request by jkozol**

### CODE REVIEW

File: devel/README.md [hunk 1]
```diff
@@ -40,6 +40,6 @@ docker compose up
 ```
 
 Access the service through the GUI:
-[http://prod.foo.redhat.com:1337/beta/](http://prod.foo.redhat.com:1337/beta/), or
+[https://prod.foo.redhat.com:1337/apps/image-builder](https://prod.foo.redhat.com:1337/apps/image-builder), or
```
Comment (teg) [inline devel/README.md:43]: Should it not be `beta/`?

Comment (jkozol) [reply devel/README.md:43]: I see that there are actually a few links that bring me to image builder. I updated this to point to `https://prod.foo.redhat.com:1337/beta/insights/image-builder` which I think makes the most sense.

Comment (teg) [reply devel/README.md:43]: Did you intend to push a new version? :)

Comment (jkozol) [reply devel/README.md]: hehe you are correct :)

**Review by teg — APPROVED** [review]

**Pull request merged**